File: kattis/easy/java/cetvrta.java
package easy;

import java.util.Scanner;

public class cetvrta {

	// gets the int with least occurences, given two are always equal
	static int getLeast(int a, int b, int c) {
		if (a == b) return c;
		if (b == c) return a;
		return b;
	}

	public static void main(String[] args) {
		try (Scanner scanner = new Scanner(System.in)) {
			int ax = scanner.nextInt(), ay = scanner.nextInt();
			int bx = scanner.nextInt(), by = scanner.nextInt();
			int cx = scanner.nextInt(), cy = scanner.nextInt();

			int x = getLeast(ax, bx, cx);
			int y = getLeast(ay, by, cy);

			System.out.println(x+" "+y);
		}
	}
}

File: kattis/easy/cpp/cetvrta.cpp
//
// Created by BBJ on 04.01.2024.
//
#include <iostream>

// gets int with least occurances, given two are always equal
int getLeast(const int& a, const int& b, const int& c) {
    if (a == b) return c;
    if (b == c) return a;
    return b;
}

int main() {
    int ax, ay; std::cin >> ax >> ay;
    int bx, by; std::cin >> bx >> by;
    int cx, cy; std::cin >> cx >> cy;

    int x = getLeast(ax, bx, cx);
    int y = getLeast(ay, by, cy);

    std::cout << x << " " << y;

    return 0;
}